tests/ttnn/nightly/unit_tests/operations/conv/test_conv2d.py

```diff
@@ -3656,3 +3656,71 @@ def test_conv2d_with_fold(
         enable_kernel_stride_folding=True,
         preprocess_weights_on_device=preprocess_weights_on_device,
     )
+
+@pytest.mark.parametrize(
+    "batch, input_channels, output_channels, input_height, input_width, weights_dtype, output_dtype, groups, kernel, stride, padding, dilation",
+    (
+        (1, 1920, 1280, 32, 32, ttnn.bfloat8_b, ttnn.bfloat16, 1, (1, 1), (1, 1), (0, 0, 0, 0), (1, 1)),
+    ),
+)
+
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 16384}], indirect=True)
```

Comment:
put this test in nightly

Reply:
Also would be nice if we could use a smaller example for this (num channels is large)
in general try to simplify test as much as possible but to still hit a repro